Ccs.Ppg.NotificationService/Services/EmailProviderService.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Ccs.Ppg.NotificationService.Model;
+using Ccs.Ppg.NotificationService.Services.IServices;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using Notify.Client;
+using Notify.Models.Responses;
+
+namespace Ccs.Ppg.NotificationService.Services
+{
+   public class EmailProviderService : IEmailProviderService
+	{
+		private readonly IConfiguration _configuration;
+		private readonly IHttpClientFactory _httpClientFactory;
+
+		public EmailProviderService(IConfiguration configuration, IHttpClientFactory httpClientFactory)
+		{
+			_configuration = configuration;
+			_httpClientFactory = httpClientFactory;
+		}
+		public async Task SendEmailAsync(EmailInfo emailInfo)
```

Comment:
add try catch in this method, refer sms method.

Reply:
Done